# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1333552602 14400
#      Wed Apr 04 11:16:42 2012 -0400
# Node ID c541f7addf2546826089219341eea757774bfc53
# Parent  f739e30494c8b7f700e520c7ed190c67ba175459
restore correct list of plot/private files in scripts/Makefile

* polynomial/module.mk: Set polynomial_PRIVATE_FCN_FILES, not
plot_PRIVATE_FCN_FILES.

diff --git a/scripts/polynomial/module.mk b/scripts/polynomial/module.mk
--- a/scripts/polynomial/module.mk
+++ b/scripts/polynomial/module.mk
@@ -1,11 +1,11 @@
 FCN_FILE_DIRS += polynomial
 
-plot_PRIVATE_FCN_FILES = \
+polynomial_PRIVATE_FCN_FILES = \
 	polynomial/private/__splinefit__.m
 
 polynomial_FCN_FILES = \
   polynomial/compan.m \
   polynomial/conv.m \
   polynomial/deconv.m \
   polynomial/mkpp.m \
   polynomial/mpoles.m \
